config/app/variables/cli/cli.json

```diff
@@ -12,8 +12,8 @@
           "options": {
             "alias": "c",
             "requiresArg": true,
-            "type": "string",
-            "describe": "The configuration for the server. The default only stores data in memory; to persist to your filesystem, use @css:config/file.json."
+            "type": "array",
+            "describe": "The configuration(s) for the server. The default only stores data in memory; to persist to your filesystem, use @css:config/file.json."
```

Comment:
(is the order meaningful?)

Reply:
It's the order on which they are sent to Components.js so depends on if the order is meaningful there, and I'm not sure about that. It shouldn't be though since it's combining all the triples. Only thing that it might impact is the order in which things get added to an array if that happens in different locations.

Reply:
…and if there are multiple overrides

Reply:
> …and if there are multiple overrides

If you have multiple overrides targeting the same resource behavior is undefined anyway. It might be that components.js even throws an error saying not to do that. In case you want to override something that already has an override, you should target that override and not the original resource.